src/targets.rs

```diff
@@ -896,6 +901,9 @@ impl Architecture {
             | LoongArch64
             | Wasm64
             | Clever(_) => Ok(PointerWidth::U64),
+            // Actually 256-bit, as are the rest of registers…
```

Comment:
I've just received a fix for this from Polygon folks - our registers will now be 64-bit.